### PR TITLE
Change dst of hashToGroup #165

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

**Comment by claucece** [comment]
cc. @chris-wood @armfazh

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -1028,8 +1028,8 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Order(): Returns
   `1000000000000000000000000000000014DEF9DEA2F79CD65812631A5CF5D3ED`
   - HashToGroup(): hash_to_ristretto255
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-ristretto255_XMD:SHA-256_ELL2_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || self.contextString
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1032]: `self.contextString` is a little hard to parse here (out of context), so how about:

```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is computed in the Setup functions.
```

(or something like that)

**Review by chris-wood — CHANGES_REQUESTED** [review]

File: draft-irtf-cfrg-voprf.md (hunk 1 above)
Comment (claucece) [reply draft-irtf-cfrg-voprf.md:1032]: Changed.

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -1028,8 +1028,9 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Order(): Returns
   `1000000000000000000000000000000014DEF9DEA2F79CD65812631A5CF5D3ED`
   - HashToGroup(): hash_to_ristretto255
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-ristretto255_XMD:SHA-256_ELL2_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || contextString, , where `contextString` is that which is
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1032]:
```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is
```

File: draft-irtf-cfrg-voprf.md [hunk 3]
```diff
@@ -1048,8 +1049,9 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Name: decaf448 {{RISTRETTO}}
   - Order(): Returns `3FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF7CCA23E9C44EDB49AED63690216CC2728DC58F552378C292AB5844F3`
   - HashToGroup(): hash_to_decaf448
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-decaf448_XMD:SHA-512_ELL2_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || contextString, , where `contextString` is that which is
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1053]:
```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is
```

File: draft-irtf-cfrg-voprf.md [hunk 4]
```diff
@@ -1069,8 +1071,9 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Order(): Returns
   `FFFFFFFF00000000FFFFFFFFFFFFFFFFBCE6FAADA7179E84F3B9CAC2FC632551`
   - HashToGroup(): P256_XMD:SHA-256_SSWU_RO\_
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-P256_XMD:SHA-256_SSWU_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || contextString, , where `contextString` is that which is
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1075]:
```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is
```

File: draft-irtf-cfrg-voprf.md [hunk 5]
```diff
@@ -1088,8 +1091,9 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Order(): Returns
   `FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFC7634D81F4372DDF581A0DB248B0A77AECEC196ACCC52973`
   - HashToGroup(): P384_XMD:SHA-512_SSWU_RO\_
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-P384_XMD:SHA-512_SSWU_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || contextString, , where `contextString` is that which is
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1095]:
```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is
```

File: draft-irtf-cfrg-voprf.md [hunk 6]
```diff
@@ -1107,8 +1111,9 @@ and curve25519. See {{cryptanalysis}} for related discussion.
   - Order(): Returns
   `1FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFA51868783BF2F966B7FCC0148F709A5D03BB5C9B8899C47AEBB6FB71E91386409`
   - HashToGroup(): P521_XMD:SHA-512_SSWU_RO\_
-    {{I-D.irtf-cfrg-hash-to-curve}} with DST
-    "VOPRF05-P521_XMD:SHA-512_SSWU_RO\_"
+    {{I-D.irtf-cfrg-hash-to-curve}} with DST:
+    "VOPRF05" || contextString, , where `contextString` is that which is
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:1115]:
```suggestion
    "VOPRF05" || contextString, where `contextString` is that which is
```

**Review by chris-wood — APPROVED** [review]
